src/main/native/GCM.c

```diff
@@ -1538,7 +1560,7 @@ jbyteArray aad, jint aadLen)
 int GCM_InitForUpdateDecrypt_core (JNIEnv *env, ICC_CTX* ockCtx, ICC_AES_GCM_CTX* gcmCtx,
 	unsigned char * key, int keyLen,
 	unsigned char * iv,  int ivLen,
-	unsigned char * aad, int aadLen,  long *updateOutlen) {
+	unsigned char * aad, int aadLen,  unsigned long *updateOutlen) {
```

Comment:
I think there's 2 spaces after the comma.

Reply:
Changed this to be a single space.